
builtin_interfaces/Time stamp
float32 attitude_radians

# TOPICS offboard/controller_input_attitude 
